src/query/execute_schema.c

```diff
@@ -2485,7 +2485,13 @@ acquire_locks_for_multiple_rename (const PT_NODE * statement)
 
       for (i = 0; i < num_names; i++)
 	{
-	  if (strcmp (name_set[i], old_name) == 0)
+	  /*
+	   * Check if old_name to be changed next is in the list of new_name that has been changed before.
+	   *   - e.g. rename table a as b, b as c, c as a;
+	   *     1. b vs b
+	   *     2. b vs c, c vs c
+	   */
+	  if (pt_str_compare (name_set[i], old_name, CASE_INSENSITIVE) == 0)
```

Comment:
In the mq_check_delete() function, pt_str_compare() is replaced with pt_user_specified_name_compare().
Here we are replacing strcmp() with pt_str_compare(CASE_INSENSITIVE). Maybe I should use pt_user_specified_name_compare()?
Was strcmp() modified considering that it was case-sensitive?

Reply:
Changed comments to clarify the reason for the change.

```
/*
 * Check if old_name to be changed next is in the list of new_name that has been changed before.
 *   - e.g. rename table a as b, B as c, C as a;
 *     1. dba.b vs dba.B
 *     2. dba.b vs dba.C, dba.c vs dba.C
 *
 *     ERROR: Unknown class "b".
 *
 * Changed comparison function to pt_str_compare() function to be case insensitive.
 */
if (pt_str_compare (name_set[i], old_name, CASE_INSENSITIVE) == 0)
  {
    found = true;
    break;
  }
```